pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# external libraries
+from sympy import Symbol
+from sympy.physics.vector import Point, Dyadic
+from sympy.physics.mechanics import ReferenceFrame
+
+# local
+from ..bodies import Body, Ground
+from ..force import Force
+
+class TestBody():
+    def setup(self):
+        self.body = Body('body')
+
+    def test_body_init(self):
+        assert self.body.name == 'body'
+        assert self.body.parent is None
+        assert self.body.child is None
+        assert isinstance(self.body.masscenter, Point)
+        assert isinstance(self.body.frame, ReferenceFrame)
+        assert isinstance(self.body.mass, Symbol)
+        assert len(self.body.inertia) == 2
+        assert isinstance(self.body.inertia[0], Dyadic)
+        assert isinstance(self.body.inertia[1], Point)
+
+    def test_body_properties_value(self):
+        name = self.body.name
+        assert self.body.masscenter == Point(name + '_MassCenter')
+        assert self.body.frame == ReferenceFrame(name + '_ReferenceFrame')
+        assert self.body.mass == Symbol(name + '_Mass')
+
+    def test_body_add_force(self):
+        assert isinstance(self.body.force_list, list())
+        assert len(self.body.force_list) == 0
+
+        self.body.add_force(Force.GRAVITY)
```

Comment:
Let me explain what I have done here, I apologize if all of this is confusing. I think I should have raised a PR for Force too.
In our yesterday's meeting tarun and I decided that instead of using Force's constructor to pass body and other paramters lets do body.add_force() which have problem that I can't access body's details like mass. I have asked for the help on gitter, until tarun sees that as a quick fix I did this implementation.
Force have methods like gravity and constants like GRAVITY which can be anything e.g. a string 'gravity' like an enum and there is no constructor of Force. There is one method and one constant for each individual force. System use Force.GRAVITY to add the force to the body and body just holds the constant.